src/query/functions/src/aggregates/aggregate_quantile_cont.rs

```diff
@@ -290,29 +290,29 @@ pub fn try_create_aggregate_quantile_function<const TYPE: u8>(
         match param {
             Scalar::Decimal(d) => {
                 let f = d.to_float64();
-                if f <= 0f64 || f >= 1f64 {
+                if f < 0f64 || f > 1f64 {
                     return Err(ErrorCode::BadDataValueType(format!(
-                        "level range between 0 to 1, got: {:?}",
+                        "level range between [0, 1], got: {:?}",
                         f
                     )));
                 }
                 f
             }
-            Scalar::Number(NumberScalar::UInt64(i)) => {
+            Scalar::Number(NumberScalar::UInt8(i)) => {

```

Comment:
Why not use `check_number` here, see `window_funnel.rs`

Reply:
done